src/pages/Quote.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Please remove this eslint disable rule.

Reply:
done